client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -26,11 +26,14 @@ export default function Dashboard(): JSX.Element {
     // loading for a split seconds until history.push works
     return <CircularProgress />;
   }
-
+  const handleTemporaryButton = () => {
+    history.push('/profile/menuitems');
+  };
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
       <CssBaseline />
       <Grid item className={classes.drawerWrapper}>
+        {<button onClick={handleTemporaryButton}>Options</button>}
```

Comment:
Maybe use an MUI button here to stay consistent with UI design

Reply:
Since this will be removed once the other features are done, I left it as is but added TODO comments for removal.